lib/config/decrypt.spec.ts

```diff
@@ -208,5 +216,92 @@ describe('config/decrypt', () => {
         CONFIG_VALIDATION,
       );
     });
+
+    it('handles EC/AES-GCM encryption', async () => {
+      GlobalConfig.set({ privateKey: toBase64(privateEc) });
+      config.encrypted = {
+        token:
+          'eyJrIjp7Imt0eSI6IkVDIiwieCI6IlM0TFhnVUJHMUNlSXVUU1ZiZEg3MHZpSjV6dUxaRFNydFJ2ZUhVTHBBYnBMTXAwQ0tyeVozWm5RN1lKQjh3N2MiLCJ5IjoiZVpQNFZzT1hXZGxRYkNWYWxSakVWNEFzTFBxYmpKdW1mR1dfTzlZbG9HM2dqbzd5enJfUXQtbXl2SG5LdkZheiIsImNydiI6IlAtMzg0In0sImkiOiJNRDVMb2hUMmh2VS9SYXA3IiwibSI6IkptMERpUmNtbEFEU2JjeVdnWmJlUXl4QkdUbU55aVlkV2NDa0tyVnhQZStTdEdMMlBwWT0ifQ==',
+      };
+
+      const res = await decryptConfig(config, 'some/def');
+      expect(res.encrypted).toBeUndefined();
+      expect(res.token).toBe('123');
+      await expect(decryptConfig(config, 'abc/defg')).rejects.toThrow(
+        CONFIG_VALIDATION,
+      );
+    });
+  });
+
+  describe('tryDecryptEc', () => {
+    it('decrypts', async () => {
+      // generate a new key pair in browser for each encryption
+      // private key will be thrown away after encryption,
+      // so only renovate can decrypt with private key
+      const browserKeyPair = await crypto.subtle.generateKey(
+        { name: 'ECDH', namedCurve: 'P-384' },
+        false,
+        ['deriveKey'],
+      );
+
+      // `d` is the private key, `x` and `y` are the public key
+      const { d, ...renovatrePublicKey } = { ...renovatePrivKey };
+
+      // import renovate public key
+      const svrPubKey = await crypto.subtle.importKey(
+        'jwk',
+        renovatrePublicKey,
+        { name: 'ECDH', namedCurve: 'P-384' },
+        false,
+        [],
+      );
+
+      // derive shared secret from our private key and renovate public key
+      const pw = await crypto.subtle.deriveKey(
+        { name: 'ECDH', public: svrPubKey },
+        browserKeyPair.privateKey,
+        { name: 'AES-GCM', length: 256 },
+        false,
+        ['encrypt'],
+      );
+
+      // always use a new IV, should always be 12 bytes
+      const iv = crypto.getRandomValues(new Uint8Array(12));
```

Comment:
Is `IV` programmer jargon? Because for me it's not clear what it means. I would prefer us to expand abbreviations like this.

Reply:
`initialization vector` or `nonce` required by `AES-GCM`

Reply:
Could you name it `initializationVector` then? Or you can use `nonce`, and explain that word in the code comment.

Reply:
I'd say IV is the term that is commonly known.
I'd write a variable as 
```
const IV = crypto.getRandomValues(new Uint8Array(12)); // Initialization vector
```
rather than the other way around.

Reply:
Lets stick with IV then!